### ISSUE TITLE
Part of main spark release

### ISSUE THREAD

**Issue by sbansal6** [EclairJS/eclairjs-node#113]
Will this ever be part of main spark distribution, like R and Python.

**Comment by doronrosenberg** [comment]
There have been no discussions regarding that, we still need to see if there is enough interest for the project

**Comment by sbansal6** [comment]
Makes sense but we are reluctant to use this in prod because its not part of main distribution..so its a chicken and egg thing :)

**Comment by EclairJS** [comment]
Hi Saurabh, this question has been on our (internal) back-burner for a while, could we discuss it further? You can reach me at fallside _a_t_ us.ibm.com.
